scene/animation/animation_mixer.cpp

```diff
@@ -2059,6 +2144,81 @@ void AnimationMixer::clear_caches() {
 	_clear_caches();
 }
 
+void AnimationMixer::set_fpslod(bool p_enabled) {
+	fps_lod = p_enabled;
```

Comment:
```suggestion
	if (fps_lod == p_enabled) {
		return;
	}
	fps_lod = p_enabled;
```
For all that run checks or notify changes, avoid unnecessary code being run